qCC/ccColorFromScalarDlg.cpp

```diff
@@ -43,13 +43,27 @@ ccColorFromScalarDlg::ccColorFromScalarDlg(QWidget* parent, ccPointCloud* pointC
 	, m_ui(new Ui::ColorFromScalarDialog)
 {
 	m_ui->setupUi(this);
-	
+	this->setAttribute(Qt::WA_DeleteOnClose, true);
```

Comment:
I made a poor suggestion here - my apologies.

Doing it this way means that you can't create one of these dialogs on the stack. (You can, but it will crash when you close it :-) ) It's safer leaving this in mainwindow.

Reply:
No problem, easy fix